Cargo.toml

```diff
@@ -39,10 +39,15 @@ members = [
 # resolver = "2"
 
 [features]
-default = ["compress", "cookies"]
+default = ["compress-br", "compress-gz", "cookies"]
 
-# content-encoding support
-compress = ["actix-http/compress"]
+# content-encoding support algorithms (gz: Gzip + Deflate, br: Brotli, zstd: Zstd)
+compress-br = ["actix-http/compress-br"]
+compress-gz = ["actix-http/compress-gz"]
+compress-zstd = ["actix-http/compress-zstd"]
+
+# Meta flag for enabling all supported compression algorithms
+compress-all = ["compress-br", "compress-gz", "compress-zstd"]
```

Comment:
As this is intended for a major release, I see no reason to have a `compress-all`. By default, I'd expect all to be enabled so we keep the previous behavior and if the user wishes to reduce it, then he/she picks each individually.

Reply:
Understood. I have remove `compress-all` flag and enable all compression algorithm by default.